### DISCUSSION TITLE
Buffer vertical Indent correspondance

### DISCUSSION THREAD

**Discussion by anthonyivol** [zed-industries/zed#10050] [Feature Requests]
I can't find how to show vertical indent correspondance, there is the show_whitespaces setting but it is not exactly what i'm looking for :

<img width="1141" alt="image" src="https://github.com/zed-industries/zed/assets/19983700/af6d5743-a6ad-4b0a-a3e2-b20b9c78038d">
vs
<img width="941" alt="image" src="https://github.com/zed-industries/zed/assets/19983700/4ca301c9-43c1-4594-a18e-4d28e4824e74">

I maybe missed something... would be nice to have this feature.

**Comment by Moshyfawn** [comment]
This feature hasn't been implemented yet. Ref: #5373.
